tests/sample/collections/test_base_collection.py

```diff
@@ -63,19 +63,3 @@ def test_as_dict(self):
         assert p.as_dict()['name'] == 'name'
         assert len(p.as_dict()['data']) == 1
         assert p.as_dict()['data'][0]['name'] == 'layer'
-
-    def test_make_default_collection(self):
```

Comment:
Should the new DEFAULT_ELEMENTS be tested instead?

Reply:
They are indirectly tested in the `test_default` for each of the collections.

I have added tests to confirm that we are also able to create empty collections.